packages/compute-baseline/src/browser-compat-data/supportStatements.test.ts

```diff
@@ -186,5 +195,147 @@ describe("statements", function () {
         assert.equal(rels.length, browser("chrome").releases.length - 10);
       });
     });
+
+    describe("supportedIn()", function () {
+      it("throws when browser is undefined", function () {
+        const cr = browser("chrome");
+        const statement = new RealSupportStatement({ version_added: "1" });
+        assert.throws(() => statement.supportedIn(cr.current()), Error);
+      });
+
+      it("throws when release does not correspond to the statement's browser", function () {
+        const statement = new RealSupportStatement(
+          { version_added: "1" },
+          browser("chrome"),
+        );
+        assert.throws(
+          () => statement.supportedIn(browser("firefox").current()),
+          Error,
+        );
+      });
+
+      it("returns supported when release is on after version_added", function () {
+        const cr = browser("chrome");
+        const unranged = new RealSupportStatement({ version_added: "100" }, cr);
+        const ranged = new RealSupportStatement({ version_added: "≤100" }, cr);
+
+        assert.equal(unranged.supportedIn(cr.version("100")).supported, true);
+        assert.equal(unranged.supportedIn(cr.version("101")).supported, true);
+        assert.equal(unranged.supportedIn(cr.current()).supported, true);
+        assert.equal(
+          unranged.supportedIn(cr.releases.at(-1) as any).supported,
+          true,
+        );
+
+        assert.equal(ranged.supportedIn(cr.version("99")).supported, null);
+        assert.equal(ranged.supportedIn(cr.version("100")).supported, true);
+        assert.equal(ranged.supportedIn(cr.version("101")).supported, true);
+        assert.equal(ranged.supportedIn(cr.current()).supported, true);
+        assert.equal(
+          ranged.supportedIn(cr.releases.at(-1) as any).supported,
+          true,
+        );
+      });
+
+      it("returns supported when release is on after version_added and before version_removed", function () {
+        const cr = browser("chrome");
+        const unranged = new RealSupportStatement(
+          { version_added: "100", version_removed: "125" },
+          cr,
+        );
+        const ranged = new RealSupportStatement(
+          { version_added: "≤100", version_removed: "125" },
+          cr,
+        );
+
+        assert.equal(unranged.supportedIn(cr.version("99")).supported, false);
+        assert.equal(unranged.supportedIn(cr.version("100")).supported, true);
+        assert.equal(unranged.supportedIn(cr.version("101")).supported, true);
+        assert.equal(unranged.supportedIn(cr.version("124")).supported, true);
+        assert.equal(unranged.supportedIn(cr.version("125")).supported, false);
+
+        assert.equal(ranged.supportedIn(cr.version("99")).supported, null);
+        assert.equal(ranged.supportedIn(cr.version("100")).supported, true);
+        assert.equal(ranged.supportedIn(cr.version("101")).supported, true);
+        assert.equal(ranged.supportedIn(cr.version("124")).supported, true);
```

Comment:
Why not the 125 assert here, so it's the same asserts for unranged and ranged? (You might remove 101 if you want to reduce the number of rows.)

Reply:
Done with 4e0110b52d7ae60a9ced2e0c19f54ea58b8fa0cb.